packages/server/src/util/muster-utils.ts

```diff
@@ -577,6 +581,108 @@ function buildUnitStats(args: {
   return unitStats;
 }
 
+/**
+ * Converts the information from a muster config to a later.js schedule
+ * @param timezone timezone of the schedule startTime
+ * @param days array of numbers in the range 1-7
+ * @param startTime schedule start time
+ * @param durationMinutes how long the muster event window lasts for
+ */
+export function getMusterSchedule(timezone: string, days: number[] | undefined, startTime: string, durationMinutes: number): later.Schedule {
+  // handle recurring muster schedule
+  if (days && days.length > 0) {
+    const start = moment.tz(startTime, 'HH:mm', timezone)
+      .utc()
+      .format('HH:mm');
+    const end = moment.tz(startTime, 'HH:mm', timezone)
+      .add(durationMinutes, 'minutes')
+      .utc()
+      .format('HH:mm');
+
+    return later.schedule(later.parse.recur()
+      .on(days)
+      .dayOfWeek()
+      .after(start)
+      .time()
+      .before(end)
+      .time());
+  }
+
+  // handle one-time muster schedule
+  const start = moment.tz(startTime, 'YYYY-MM-DDTHH:mm:ss', timezone).utc();
+  const end = moment.tz(startTime, 'YYYY-MM-DDTHH:mm:ss', timezone)
+    .add(durationMinutes, 'minutes')
+    .utc()
+    .format('HH:mm');
+
+  return later.schedule(later.parse.recur()
+    .on(start.year())
+    .year()
+    .on(start.dayOfYear())
+    .dayOfYear()
+    .after(start.format('HH:mm'))
+    .time()
+    .before(end)
+    .time());
+}
+
+/**
+ * Gets the total required number of musters for a unit based on all configs for the unit
+ * over a given period of time.
+ * @param unitConfigs an array of config objects to check the count for the unit
+ * @param fromDate start of date range to get muster count for
+ * @param toDate cutoff for date range to get muster count for
+ */
+export function getUnitRequiredMusterCount(unitConfigs: object[] | undefined, fromDate: moment.Moment, toDate: moment.Moment): number {
```

Comment:
Do we have a more specific type than `object` we can use for `unitConfigs`? Can those be `MusterConfiguration[]`?